.github/workflows/ci.yml

```diff
@@ -231,6 +231,44 @@ jobs:
       - name: Run yarn test262
         run: exit 0 # TODO remove test262 from required tests for CI
 
+  test-hermes:
+    name: test-hermes
+
+    # begin macro
+
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ubuntu-latest]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      # without this, setup-node errors on mismatched yarn versions
+      - run: corepack enable
+
+      - name: Use Node.js 22.x
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22.x
+          cache: yarn
+
+      - name: Echo node version
+        run: node --version
```

Comment:
this info is printed automatically by the runner. if there's another reason why it's here, pls add a comment

Reply:
was going to remove this too, but kept in line with our other 6 jobs

done in
- https://github.com/endojs/endo/pull/2458